### PR TITLE
Use present to check ENV variables

### PR DESCRIPTION
**Pull request by scytacki**
This should fix a delay seen on portals without google optimizer configured

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -29,7 +29,7 @@ def pundit_user_not_authorized(exception)
       render :text => "<div class='flash_error'>#{error_message}</div>", :status => 403
     else
       if current_user
-        if ENV['RESEARCHER_REPORT_ONLY']
+        if ENV['RESEARCHER_REPORT_ONLY'].present?
```
Comment (dougmartin) [inline app/controllers/application_controller.rb]: This is a boolean, so couldn't it be set to false?

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -294,7 +294,7 @@ def after_sign_in_path_for(resource)
 
     redirect_path = view_context.current_user_home_path
 
-    if ENV['RESEARCHER_REPORT_ONLY']
+    if ENV['RESEARCHER_REPORT_ONLY'].present?
```
Comment (dougmartin) [inline app/controllers/application_controller.rb]: This is a boolean, so couldn't it be set to false?

File: app/views/report/learner/index.html.haml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 
-- if ENV['RESEARCHER_REPORT_ONLY']
+- if ENV['RESEARCHER_REPORT_ONLY'].present?
```
Comment (dougmartin) [inline app/views/report/learner/index.html.haml]: This is a boolean, so couldn't it be set to false?

File: config/environments/production.rb [hunk 4]
```diff
@@ -23,7 +23,7 @@
 
   # Use a different logger for distributed setups
   # config.logger = SyslogLogger.new
-  if ENV['RAILS_STDOUT_LOGGING']
+  if ENV['RAILS_STDOUT_LOGGING'].present?
```
Comment (dougmartin) [inline config/environments/production.rb]: This is a boolean, so couldn't it be set to false?

File: db/migrate/20151030132107_create_material_properties.rb [hunk 5]
```diff
@@ -62,7 +62,7 @@ def down
   # New models that have added to app/models/** expect that their tables exist.
   # Loading those new models here, will throw exceptions. (As happened to me)
   def reindex
-    if ENV["REINDEX_SOLR"] # We would need to set this explicitly to re-index
+    if ENV["REINDEX_SOLR"].present? # We would need to set this explicitly to re-index
```
Comment (dougmartin) [inline db/migrate/20151030132107_create_material_properties.rb]: This is a boolean, so couldn't it be set to false?

**Comment by scytacki** [comment]
@dougmartin I think I've address those issues.

**Comment by scytacki** [comment]
Turns out that I missed one boolean. And this also broke the production docker build because the BoolENV wasn't loaded yet before it was used.

**Pull request merged**